spec/spec_helper.rb

```diff
@@ -26,8 +26,11 @@ def page(options={})
 end
 
 def post(options={})
-  page = Jekyll::Post.new site, CONFIG_DEFAULTS["source"], "", "2015-01-01-post.md"
-  page.data = options
+  site = site()
```

Comment:
I need to use `site` twice in `config` and wanted to avoid creating two objects. Does this make sense? Is this ok?